MNE/utils/mp/adaptivemp.cpp

```diff
@@ -209,7 +209,7 @@ QList<QList<GaborAtom> > AdaptiveMp::matching_pursuit(MatrixXd signal, qint32 ma
         k = 0;
         s = sample_count;
         p = floor(sample_count / 2);
-        j = floor(log2(sample_count));//log(sample_count) / log(2));
+        j = floor(log10(sample_count));//log(sample_count) / log(2));
```

Comment:
Okay than we should definitvely divide by log10(2), i.e., log10(sample_count) /  log10(2), to still compute the logarithmus dualis